src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -890,8 +894,8 @@ def clone_status(self, **kwargs):
         try:
             with open_volume(self, volname) as fs_handle:
                 with open_group(fs_handle, self.volspec, groupname) as group:
-                    with open_subvol(self.mgr, fs_handle, self.volspec, group, clonename, SubvolumeOpType.CLONE_STATUS) as subvol:
-                        status = self._get_clone_status(fs_handle, group, subvol)
+                    with open_subvol(self.mgr, fs_handle, self.volspec, group, clonename, SubvolumeOpType.CLONE_STATUS) as subvolume:
```

Comment:
This hunk should be in commit 2b203041a93b96cb300aa390c90b4c40f58c8532.

Reply:
Done.